Fixes #1610 - EGLL SMR Update

epicdemon99 opened the pull request:
# Summary of changes

SMR updated to be ready for updated charts/NOTAMs as of 1904

# Screenshots (if necessary)

Whole SMR http://prntscr.com/n2vqbo
SMR (currently commented out - due lack of scenery) http://prntscr.com/mt3vd9

hsugden commented:
Which screenshot is which? Would just be simplest to provide a link to what the SMR will look like if these changes are approved, rather than 3 versions :) 

Why is some of it commented out?

epicdemon99 commented:
> Which screenshot is which? Would just be simplest to provide a link to what the SMR will look like if these changes are approved, rather than 3 versions :)
> 
> Why is some of it commented out?

Updated it to (hopefully) be more clear - the first link is the actual SMR.
The second I have updated it based on the supposed new stands layout, but there is no point updating as there is no scenery that is accurate to the charts yet.
